collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,26 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
```

Comment:
hi, @andyxning 
I copied [the code from Kubernetes core repo](https://github.com/kubernetes/kubernetes/blob/4bfe9b1a568be137f72cf5dfc8bc2c1f13874548/pkg/api/helper/helpers.go#L561). This function is also used by [pv controller](https://github.com/kubernetes/kubernetes/blob/402e48b072b32ae79f59074d2fc67856f5a58849/pkg/controller/volume/persistentvolume/pv_controller.go).